src/precompiletools.jl

```diff
@@ -0,0 +1,51 @@
+using PrecompileTools: @setup_workload, @compile_workload
+@setup_workload begin
+    bad_bad_code = """# Some ill-formatted julia code to fire-up Runic.jl!
+    # This is simply taken from runstone
+    print( "chaotic use of spaces (yes there's one space at the end of the line") 
+      
+    \tprint( "chaotic use of tabs")\t 
```

Comment:
This is a good reminder -- currently tabs are not replaced by spaces :)

Reply:
https://github.com/fredrikekre/Runic.jl/commit/bdbe60e230b0c35a81320239dfe8725848bcb11a